src/components/Post/Post.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './Post.scss';
+import { User } from '../User/User';
+import { PostShape } from '../shapes/PostShape';
+import { CommentList } from '../CommentList/CommentList';
```

Comment:
На майбутнє хорошою практикою є якось розділяти й сортувати тут. Хоча як саме найкраще я ще сам не знаю))
```suggestion
import React from 'react';

import { User } from '../User/User';
import { PostShape } from '../shapes/PostShape';
import { CommentList } from '../CommentList/CommentList';

import './Post.scss';
```